Core/Systems/_Essentials/LatiosWorldUnmanagedSystem.cs

```diff
@@ -111,7 +111,10 @@ public void OnDestroy(ref SystemState state)
         {
             m_impl->m_worldUnmanaged.EntityManager.CompleteAllTrackedJobs();
 
-            (m_impl->m_managedStructStorage.Target as ManagedStructComponentStorage).Dispose();
+            if (m_impl->m_managedStructStorage.IsAllocated)
+            {
+                (m_impl->m_managedStructStorage.Target as ManagedStructComponentStorage)?.Dispose();
+            }
```

Comment:
This is one I discovered in 0.7 but forgot I needed to backport. Thanks for the reminder!